docs/PRIVACY.md

```diff
@@ -0,0 +1,53 @@
+
+By using a Service, you agree to the collection and use of data in accordance with these policies. We do not disclose, share, sell, or transfer any information about Service users unless required by law or for law enforcement.
+
+## Data We Collect
+
+When you use our Service we:
+
+* Do not collect data that identifies you personally (“Personal Data”) unless you choose to provide that information to us.
+* Do collect information that does not identify you personally, including data collected automatically either generated by the use of the Service or from the Service infrastructure (“Usage Data”). Usage Data may include location information, which we may use to provide you with more relevant information or language choices.
+
+Our Apps are not intended to be used by or to solicit information of any kind from children under age 13.
+
+## How We Use Data
+
+The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
+
+## Third-Party Apps and Services
+
+Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see [third party tools][8]). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may stop push notifications by adjusting your device settings.
+
+## Disclaimer
+
+THE MATERIALS EMBODIED IN THIS SERVICE ARE PROVIDED “AS-IS” AND WITHOUT WARRANTY OF ANY KIND, EXPRESSED, IMPLIED OR OTHERWISE, INCLUDING WITHOUT LIMITATION, ANY WARRANTY OF FITNESS FOR A PARTICULAR PURPOSE. IN NO EVENT SHALL THE WORLD HEALTH ORGANIZATION BE LIABLE TO YOU OR ANYONE ELSE FOR ANY DIRECT, SPECIAL, INCIDENTAL, INDIRECT OR CONSEQUENTIAL DAMAGES OF ANY KIND, OR ANY DAMAGES WHATSOEVER, WHETHER OR NOT ADVISED OF THE POSSIBILITY OF SUCH LOSS, HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, ARISING OUT OF OR IN CONNECTION WITH THE POSSESSION, USE OR PERFORMANCE OF THIS SERVICE.
+
+## Data Retention and Transfer
+
+We retain and use Personal Data and Usage Data only as long as is necessary for the purposes set out in our Privacy Policy and this Privacy Notice, or to the extent necessary to comply with legal obligations, resolve disputes, or enforce legal agreements and policies. We transfer the data, including Personal Data, to the U.S.A. for processing and storage, and by using or submitting information via the Service you consent to such transfer.
```

Comment:
Content mismatch: "Useage Data" in the [current privacy.pdf](https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf)

The original word "useage" does [appear to be a misspelling](https://linguistlist.org/ask-ling/message-details1.cfm?asklingid=200312267) in American and British English. But I would confirm even typos with legal…